addons-l10n/en/mediarecorder.json

```diff
@@ -2,19 +2,20 @@
   "mediarecorder/record": "Record",
   "mediarecorder/stop": "Stop Recording",
   "mediarecorder/option-title": "Record Options",
-  "mediarecorder/record-duration": "Record Duration (in seconds)",
-  "mediarecorder/start-delay": "Start Delay (in seconds)",
+  "mediarecorder/record-duration": "End recording after:",
+  "mediarecorder/start-delay": "Start recording after:",
+  "mediarecorder/seconds": "seconds",
   "mediarecorder/starting-in": "Starting in {secs}...",
-  "mediarecorder/record-audio": "Include project sounds",
-  "mediarecorder/record-mic": "Include sounds from microphone",
+  "mediarecorder/record-audio": "Record project audio",
+  "mediarecorder/record-mic": "Record microphone audio",
   "mediarecorder/record-audio-description": "This does not include Text-to-Speech.",
-  "mediarecorder/record-after-flag": "Do not start recording until the Green Flag is clicked",
-  "mediarecorder/record-until-stop": "Stop recording after the project has stopped",
-  "mediarecorder/record-description": "Record the stage as a {extension} file. You can save it to your computer after the recording is finished.\nNote: variable and list monitors will not be visible.",
+  "mediarecorder/record-after-flag": "Wait until green flag is clicked",
+  "mediarecorder/record-until-stop": "End recording when project is stopped",
+  "mediarecorder/record-description": "Record a video of the stage and save it to your computer as a {extension} file after the recording is finished.\nNote: variable and list monitors will not be visible in the recording.",
   "mediarecorder/start": "Start",
   "mediarecorder/cancel": "Cancel",
   "mediarecorder/added-by": "Added by Scratch Addons browser extension",
   "mediarecorder/click-flag": "Waiting...",
-  "mediarecorder/click-flag-description": "Click the green flag to start recording. Click this button to stop.",
+  "mediarecorder/click-flag-description": "Click the green flag to start recording. Click here to abort the recording.",
```

Comment:
You changed "stop" to "end" in the popup and "abort" in this tooltip. The same word should be used everywhere - I prefer "stop", but "end" is also an option.

Reply:
Correct me if I'm wrong, but I think this tooltip appears when you hover over the "Waiting..." button when the recorder is waiting for the green flag to be clicked before recording. If you click "Waiting...", it cancels the recording. That's why I used the term "abort".

Reply:
About the change of "stop" to "end", would you like if I changed "End recording after:" to "Stop recording after:" and/or do the same with "End recording when project is stopped"?

I used the term "end" in that second one just to not say "stop" twice, but it's fine either way.